src/components/Activity.jsx

```diff
@@ -64,9 +64,19 @@ export default class Activity extends Component {
     if (props.participant_counter != null)
       participants = ` (${props.participant_counter})`;
 
+    let day = new Date();
+    let pride = day.getMonth() == 5;
+
     let className = "activity";
     if (props.active) {
       className += " active";
+      if (pride) {
+        className += " activeRainbow";
+      }
+    }
+
+    if (pride) {
+      className += " rainbow";
```

Comment:
Could you add some comments on what the effects are of applying the classnames? We know, and the code reads fine, but it might help for other people to say something like "If we are in pride month, the activities should gain some rainbow effects or other pride related thigns."